src/templates/cmp.js

```diff
@@ -241,5 +244,8 @@
     }
   });
 
+  //migrate consent to new cookie
+  window.__cmpapi('_migrateConsent');
```

Comment:
Looks to me as if this is always called, regardless if the consent has already been migrated or not

Reply:
Indeed, the check if a migration is actually necessary happens within the `_migrateConsent` method.

The reason to do it like this are the following:
1) by calling the method this way, it is handled like any other api method call, therefore automatically working in both, iFrame and no-iFrame cases, also not having to worry about the evaluation of query params (in the iFrame case, localStorage is read in iframe.html and consent status is passed to cmpapi.js)
2) calling `getTCData` here would not indicate if a migration is necessary or not, as it is backwards-compatible with the old consent cookie and would just return me the consent status, independent of which cookie/localStorage it came from
3) executing the logic behind this api call directly in cmpapi.js would not allow the logging for the migration event to work, as the queue-handling is happening in cmp.js. Therefore the migrateConsent log event would never be forwarded to any listener, making it invisible and therefore not debuggable on a real device (Sofia Digital case)